src/lwc/geTemplateBuilderFormFields/geTemplateBuilderFormFields.js

```diff
@@ -188,20 +191,52 @@ export default class geTemplateBuilderFormFields extends LightningElement {
             const lightningAccordion = this.template.querySelector('lightning-accordion');
             lightningAccordion.activeSectionName = [...objectMappingsWithMissingRequiredFields];
 
-            this.hasErrors = true;
             this.errors = [...missingRequiredFieldMappings];
 
+            this.pageLevelError = {
```

Comment:
We seem to be missing the required fields error messages now in the template builder. I think we might have scenarios where we have broken field references and missing required fields, in that case we want to prevent saving a template that does not have those fields (`Account1Imported`, `Account1Name`, `Contact1Imported`, `Contact1LastName`)

Reply:
@voduyemi @psadie-sfdo added the `errors` property to the page level object. Let me know if this is what you were referring to. https://github.com/SalesforceFoundation/NPSP/pull/5845/commits/acb324dcb4b51eddc4188db2561963cb2c057f70 and https://github.com/SalesforceFoundation/NPSP/pull/5845/commits/425bf8fdea66b50b770ac3ff1045678dd5899440

Reply:
@voduyemi i created a brand new scratch org off of master. This scenario where you remove an "always required" field like Account1Imported, that's not being blocked in the master branch currently either. so you might double check there. idk if we regressed on that at some point but in the master branch I was able to remove all the fields you listed above from the template without being blocked.

Reply:
Out of the box, technically the only "required" field in template builder is Donation Donor.

Reply:
I took a closer look at this, and I think I was wrong, it's working like it should, if you take all those fields off the template, you should get this error message: 
![image](https://user-images.githubusercontent.com/20406258/93788856-5ed49600-fbee-11ea-9ef8-f249948006b3.png)